src/OpenFeature.Extensions.Hosting/OpenFeature.Extensions.Hosting.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Nullable>enable</Nullable>
+    <TargetFrameworks>netstandard2.0;net6.0;net7.0;net8.0;net462</TargetFrameworks>
```

Comment:
Should .NET 7 be included as TFM? [.NET 7 is out of official support](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core)